client/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/ignore-files/ for more about ignoring files.
```

Comment:
I don't see why you'd need more than one .gitignore in your files, the one in the root should suffice

Reply:
b077f7bf082451873594bd4f524b1f29276e2a07

Reply:
c5678d9b577e7cb17bbe95ab0cde55987510dd51